apps/omg_watcher_info/lib/omg_watcher_info/api/account.ex

```diff
@@ -30,8 +30,8 @@ defmodule OMG.WatcherInfo.API.Account do
   @doc """
   Gets all utxos belonging to the given address.
   """
-  @spec get_utxos(OMG.Crypto.address_t()) :: list(%DB.TxOutput{})
-  def get_utxos(address) do
-    DB.TxOutput.get_utxos(address)
+  @spec get_utxos(Keyword.t()) :: Paginator.t()
```

Comment:
We lose some detail from `list(%DB.TxOutput{})` -> `Paginator.t()`. Can we do this?

```elixir
@spec get_utxos(Keyword.t()) :: Paginator.t(%DB.TxOutput{})
```

Need to change `Paginator.t()` to

```elixir
@type t(data_type) :: %__MODULE__{
  data: list(data_type),
  data_paging: %{limit: pos_integer(), page: pos_integer()}
}
```

and update a few more places that use `:: Paginator.t()` but shouldn't be too many (I count 4).